### PR TITLE
Feature/asb 1953 ni home page

### PR DESCRIPTION
**Pull request by emil-placheta**
## Describe your changes

Ticket: https://pins-ds.atlassian.net/jira/software/c/projects/ASB/boards/166?selectedIssue=ASB-1953

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/pages/index/utils/get-page-data.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const { mapTitles } = require('../../_utils/map-titles');
+const { haveYourSayGuideSubdirectory } = require('../../have-your-say-guide/config');
+const { processGuideSubdirectory } = require('../../process-guide/config');
+mapTitles;
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/index/utils/get-page-data.js:4]: remove

File: packages/forms-web-app/src/sass/components/section/masthead.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.masthead {
+    background-color: #00847f;
+    margin-bottom: 30px;
+
+    &--full-width-teal-background {
+        clip-path: inset(0 -100vmax);
+        box-shadow: 0 0 0 100vmax #00847f;
+    }
+
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/sass/components/section/masthead.scss:9]:
```
&--bg {
    &-teal {
        background-color: #00847f;
    }
}

&--full-width {
    clip-path: inset(0 -100vmax);

    &.masthead--bg {
        &-teal {
            box-shadow: 0 0 0 100vmax #00847f;
        }
    }
}
```

File: packages/forms-web-app/src/routes/index.js [hunk 3]
```diff
@@ -22,7 +22,9 @@ const { registerSubdirectory } = require('../pages/projects/register/config');
 const { apiSubdirectory } = require('../api/config');
 const { processGuideRouter } = require('../pages/process-guide/router');
 const { haveYourSayGuideRouter } = require('../pages/have-your-say-guide/router');
+const { indexRouter } = require('../pages/index/router');
 
+router.use('/', indexRouter);
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/routes/index.js:27]: remove '/'

File: packages/forms-web-app/src/sass/components/ui/chevron-card.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    &__link {
+        display: inline-block;
+        max-width: 90%;
+    }
+
+    &__link::after {
+        border-style: solid;
+        border-width: 0.2rem 0.2rem 0 0;
+        content: '';
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/sass/components/ui/chevron-card.scss:23]: consistent content property

File: packages/forms-web-app/src/pages/_utils/map-titles.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
-const mapTitles = (pageHeading, pageTitle) => ({
-	pageHeading,
-	pageTitle
-});
+const mapTitles = (pageHeading, pageTitle) => {
+	if (!pageTitle) pageTitle = pageHeading;
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/pages/_utils/map-titles.js:2]: Look into re-assignment of argument. Is this ok?

**Pull request closed without merging**